src/main/scala/is/hail/utils/richUtils/RichSQLContext.scala

```diff
@@ -1,29 +0,0 @@
-package is.hail.utils.richUtils
-
-import is.hail.utils._
-import org.apache.spark.rdd.RDD
-import org.apache.spark.sql.functions._
-import org.apache.spark.sql.{Row, SQLContext}
-
-class RichSQLContext(val sqlContext: SQLContext) extends AnyVal {
-  def readParquetSorted(dirname: String, selection: Option[Array[String]] = None): RDD[Row] = {
-    val parquetFiles = sqlContext.sparkContext.hadoopConfiguration.globAll(Array(dirname + "/*.parquet"))
-    if (parquetFiles.isEmpty)
-      return sqlContext.sparkContext.emptyRDD[Row]
-
-    var df = sqlContext.read.parquet(dirname + "/part-*")
-    selection.foreach { cols =>
-      df = df.select(cols.map(col): _*)
-    }
-
-    val rdd = df.rdd
-
-    val oldIndices = rdd.partitions
-      .map { p => getParquetPartNumber(partitionPath(p)) }
```

Comment:
I wish we could get rid of `partitionPath`, too, but it is still used by `textFilesLines`.